package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apiaryio/console-proxy",
-  "version": "0.0.7",
+  "version": "0.0.0-semantically-released",
```

Comment:
You're right.
However given that it's my birthday the day after tomorrow, can I take this as my birthday gift?